src/entity/user/Admin.java

```diff
@@ -0,0 +1,48 @@
+package entity.user;
+
+import entity.bookType.Audiobook;
+import entity.bookType.Book;
+import entity.bookType.EBook;
+import entity.category.Category;
+import entity.category.Subcategory;
+import entity.company.Company;
+import entity.loan.Loan;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+public class Admin extends User{
```

Comment:
even if you created a singleton without setters, however it inherits from USer and access to setters (for password for example) are available and somebody can change your instance